components/icons/ArrowIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from "react"
+
+interface Props {
```

Comment:
I usually use the naming convention of `ArrowIconProps` mostly because if you need to import this type you'll need to rename it and that's always undesirable